NEWS.txt

```diff
@@ -3,58 +3,40 @@ Release Notes
 =============
 
 
-Theano 0.10.0beta2 (7th of September, 2017)
-===========================================
+Theano 0.10.0beta3 (20th of September, 2017)
+============================================
 
 This release contains new features, improvements and bug fixes to prepare the upcoming release candidate.
 
 We recommend that every developer updates to this version.
 
 Highlights:
- - Support NumPy ``1.13``
- - Support pygpu ``0.7``
- - Added conda recipe
- - Optional faster optimization step with new destroy handler
- - Added documentation for RNNBlock
- - Bug fixes, crash fixes, warning improvements and documentation updates
+ - conda packages for releases now available and updated in our own conda channel ``mila-udem``
```

Comment:
Add something like:

to use them for the beta packages, use this command "conda install -c mila-udem/label/pre theano pygpu"